apps/user-office-backend/src/datasources/postgres/AdminDataSource.ts

```diff
@@ -44,6 +44,8 @@ const seedsPath = path.join(dbPatchesFolderPath, 'db_seeds');
 
 @injectable()
 export default class PostgresAdminDataSource implements AdminDataSource {
+  protected autoUpgradedDBReady = true;
```

Comment:
It looks like we've got two initialisers for this property, both here and in the constructor. Does it need to be set twice? Could we just set it false here and remove the other?

Reply:
I had this here originally as if it was using `PostgresAdminDataSource` not `PostgresAdminDataSourceWithAutoUpgrade` this would never be set to `true` but I think it doesn't matter. I'll remove it.